shared/clinical-code-sets/patient/did-not-attend/1/did-not-attend.ctv3.txt

```diff
@@ -0,0 +1,149 @@
+62S4.	Maternal P/N exam. defaulted
+9N4..	Failed encounter
+9N4..	Other failed encounter
+9N41.	Did not attend - reason given
+9N41.	DNA - Did not attend - reason given
+9N42.	Did not attend - no reason
+9N42.	Patient defaulted from follow up
+9N42.	Patient defaulted from follow-up
+9N43.	Patient too late
+9N44.	Did not attend - appointment mix-up
+9N44.	DNA - appointment mix-up
+9N45.	Patient walked out
+9N47.	Appointment cancelled by patient
+9N47.	Appt cancelled by patient
+9N47.	PCA - Appt cancelled by patient
+9N4B.	Poor attender
+9N4C.	Failed encounter - no answer when rang back
```

Comment:
I thought of these as more specific versions of 'failed encounter' and therefore are a type of DNA. E.g. a patient 'fails' their encounter by not showing up, and then the GP tries to call them. These codes came from the PI and were originally from ClinicalCodes.org. Are you happy if I explain in the README that this code set is for LH003, but it would be a good idea to review it before using it?

Reply:
README file updated.